gqlalchemy/query_builders/memgraph_query_builder.py

```diff
@@ -81,30 +81,39 @@ def load_csv(self, path: str, header: bool, row: str) -> "DeclarativeBase":
         return self
 
     def construct_subgraph_query(
-        self, node_label: Optional[str] = None, relationship_types: Optional[Union[str, Iterable[str]]] = None
+        self,
+        node_labels: Optional[Union[str, Iterable[str]]] = None,
+        relationship_types: Optional[Union[str, Iterable[str]]] = None,
+        relationship_direction: Optional[RelationshipDirection] = RelationshipDirection.RIGHT,
     ) -> str:
-        """Constructs a MATCH query defining a subgraph using zero or one node labels and relationship types.
+        """Constructs a MATCH query defining a subgraph using one or two node labels and relationship types.
 
         Args:
-            node_label: A string representing a node label to be used in this subgraph.
+            node_labels: If string, used for both sides of the relationship, if list of two, used
             relationship_types: A string or iterable of types of relationships used in the subgraph.
 
         Returns:
             a string representing a MATCH query for a path with given node labels and relationship types.
         """
-        node = CypherNode(node_label)
+        if not node_labels or isinstance(node_labels, str):
+            node1 = node2 = CypherNode(node_labels)
```

Comment:
I would go with `first_node_label`, `second_node_label` , since it's more clear and since we did not use numbers so far. If you have something better in mind, even better.

Reply:
it's not just the label, it's the entire node including parentheses, e.g. `(:LABEL)`

Reply:
Then `first_node` and `second_node` or leave it as it is 😄 I will approve the PR now.

Reply:
This request has been implemented.